ceph/ceph.py

```diff
@@ -1846,7 +1846,9 @@ def install_ceph_ansible(self, rhbuild, **kw):
                 raise CommandFailed(err)
 
         if kw.get('upgrade'):
-            self.exec_command(sudo=True, cmd='yum update metadata')
+            self.exec_command(sudo=True,
+                              cmd='yum update metadata',
```

Comment:
actually it is meta instead of metadata, for some reason it stayed on for so long :)

Reply:
We had a same problem in teuthology, we made added `metadata`. https://github.com/ceph/teuthology/blob/master/teuthology/task/internal/redhat.py#L101

Reply:
this too. https://github.com/red-hat-storage/cephci/blob/master/tests/misc_env/install_prereq.py#L75

Reply:
@vasukulkarni  @rakeshgm Just to be clear, is it meta or metadata?

Reply:
I would suggest to skip running this command if its rhel-8. both `meta` and `metadata` fails.

Reply:
```sudo yum udpate meta 
Updating Subscription Management repositories.
No such command: udpate. Please use /bin/yum --help
It could be a YUM plugin command, try: "yum install 'dnf-command(udpate)'"
```

Reply:
```sudo yum udpate metadata 
Updating Subscription Management repositories.
No such command: udpate. Please use /bin/yum --help
It could be a YUM plugin command, try: "yum install 'dnf-command(udpate)'"```

Reply:
yes i couldnt find any man page to 'yum update meta' cmd ..pl provide if it exists